balanceRobot.ino

```diff
@@ -90,4 +78,23 @@ void loop(void) {
   } 
 }
 
-void read3D
+void read3D {
+  sensors_event_t accelEvent;
+  accel.getEvent(&accelEvent);
+    //Read every 10th data from 3D axis sensor
+  if(count == 200){
+      x = accelEvent.acceleration.x;
+      y = accelEvent.acceleration.y;
+      z = accelEvent.acceleration.z;
+    Serial.print("Accelerometer: ");
```

Comment:
Is there a way to print all data in one line? `Serial.printf(F("Value: %3d\n"), value);`.